### PR TITLE
Many small fixes

### PR DESCRIPTION
**Pull request by L5478**
- Save system feedback
- Pause menu background
- Play instrument: airguitar spam not possible anymore
- Rock splash removed
- Bridge stays up
- Oracle banjo disappears
- Deleted debug.logs

### CODE REVIEW

File: Assets/_Scenes/Scene_OpenWorld/UITextures/pause_mensu.png.meta [hunk 1]
```diff
@@ -0,0 +1,115 @@
+fileFormatVersion: 2
```
Comment (RyyzQ) [inline Assets/_Scenes/Scene_OpenWorld/UITextures/pause_mensu.png.meta]: pause_men*s*u.

(filename)

File: Assets/_Scenes/General/Scripts/Save/SaveSystem.cs [hunk 2]
```diff
@@ -101,7 +84,7 @@ public void Save()
         bf.Serialize(file, data);
         file.Close();
 
-        Debug.Log("Save succeed");
+        saveInfoTextAnimator.SetTrigger("save");
```
Comment (RyyzQ) [inline Assets/_Scenes/General/Scripts/Save/SaveSystem.cs]: Are we sure saving will always be done successfully?

File: Assets/_Scenes/Scene_OpenWorld/Scripts/Dialogue/DialogueTrigger.cs [hunk 3]
```diff
@@ -17,12 +19,12 @@ public class DialogueTrigger : MonoBehaviour
     private bool itemReceived;
     private int dialoguesCount;
 
-
     public int DialogueNumber { get => dialogueNumber; set => dialogueNumber = value; }
```
Comment (mrtsven) [inline Assets/_Scenes/Scene_OpenWorld/Scripts/Dialogue/DialogueTrigger.cs]: Move this up to the public variables

File: Assets/_Scenes/Scene_OpenWorld/Scripts/Dialogue/DialogueTrigger.cs [hunk 4]
```diff
@@ -17,12 +19,12 @@ public class DialogueTrigger : MonoBehaviour
...
 
     private void Start()
     {
         audioSource = GetComponentInChildren<AudioSource>();
+        Inventory.Instance.onItemChangedFallback += RemoveGameobjectFromWorld;
```
Comment (RyyzQ) [inline Assets/_Scenes/Scene_OpenWorld/Scripts/Dialogue/DialogueTrigger.cs]: what

File: Assets/_Scenes/Scene_OpenWorld/Scripts/SimonSays/SimonSaysTarget.cs [hunk 5]
```diff
@@ -60,7 +60,7 @@ public void PlayFalling()
     public void PlaySplash()
```
Comment (mrtsven) [inline Assets/_Scenes/Scene_OpenWorld/Scripts/SimonSays/SimonSaysTarget.cs]: Give the method a proper name as well.

File: Assets/_Scenes/Scene_OpenWorld/UITextures/pause_mensu.png.meta (hunk 1 above)
Comment (L5478) [reply Assets/_Scenes/Scene_OpenWorld/UITextures/pause_mensu.png.meta]: yeah... art team, what you can do

File: Assets/_Scenes/General/Scripts/Save/SaveSystem.cs (hunk 2 above)
Comment (L5478) [reply Assets/_Scenes/General/Scripts/Save/SaveSystem.cs]: I guess we could, but I have no idea what to do if binary formatter does not work :E

Comment (RyyzQ) [reply Assets/_Scenes/General/Scripts/Save/SaveSystem.cs]: If it doesn't, we just cry.

**Pull request merged**